CHANGES.md

```diff
@@ -1,276 +1,2504 @@
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2241'>DDF-2241</a> - Create a Historian user for all history actions
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2572'>DDF-2572</a> - Create ProcessingFramework Default (In Memory) Implementation
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2573'>DDF-2573</a> - Create ProcessingPostIngestPlugin
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2695'>DDF-2695</a> - Add taxonomy attributes for better normalization
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2715'>DDF-2715</a> - Make Point of Contact Read-only
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2783'>DDF-2783</a> - Update thumbnails to allow animation in the summary view
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2791'>DDF-2791</a> - Update the SolrFilterDelegate to support empty isEqualTo and isLike queries
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2811'>DDF-2811</a> - Invalid data in workspace metacards causes workspaces page to fail
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2812'>DDF-2812</a> - Bad workspace data can be manually inserted causing unreliable workspace view
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2815'>DDF-2815</a> - As an integrator I would like to be able to specify a Geospatial query by Universal Transverse Mercator so that I can express Geospatial queries via Universal Transverse Locator through web services
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2817'>DDF-2817</a> - HTTP responses from DDF should include a CSP header
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2827'>DDF-2827</a> - Change the default authentication mechanism in DDF to the IdP
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2849'>DDF-2849</a> - CQL endpoint does not respect Accept-Encoding header
+	</li>
+</ul>
+
+<h3>Technical Debt</h3>
+<ul>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2436'>DDF-2436</a> - Ignored Catalog Framework unit tests need to be updated 
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2451'>DDF-2451</a> - Deprecate all the legacy attributes and add a compatibility plugin
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2673'>DDF-2673</a> - Add iTests to test different query modes of CachingFederationStrategy
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2731'>DDF-2731</a> - Move PropertiesFileReader where it can be re-used
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2798'>DDF-2798</a> - There are several possible null pointers within the ECP handlers
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2821'>DDF-2821</a> - Check audit messages for special characters and encode them before writing the audit
+	</li>
+</ul>
+
+<h3>Spike</h3>
```

Comment:
❗️ Do we really need to include spikes in the change logs? Those do not (and should not) involve code changes.